src/patterns/molecules/error/error.scss

```diff
@@ -0,0 +1,45 @@
+@use "../../../shared-styles/normalize"; // Always used first
+@use "../../../shared-styles/baseline";
+@use "../../../shared-styles/grid";
+@use "../../../shared-styles/media-query";
+@use "../../../shared-styles/rem";
+@use "../../../shared-styles/scale";
+@use "../../../shared-styles/size";
+@use "../../../shared-styles/space";
+@use "../../../shared-styles/typography";
+
+.error {
+  text-align: center;
+  @include size.max-inline-size(grid.$max-inline-size);
+  @include space.margin(auto, inline);
+  @include space.padding(grid.$column_gap, inline);
+  @include space.block($start: rem.calculate(baseline.for-units(12)), $end: rem.calculate(baseline.for-units(12)));
+  @include media-query.mq($from: medium) {
+    @include space.block($start: rem.calculate(baseline.for-units(20)), $end: rem.calculate(baseline.for-units(20)));
+  }
+
+  @supports (display: flex) {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  > :last-child {
+    @include space.margin(0, block-end);
+  }
+
+}
+
+.error__image {
+  @include space.block($end: baseline.for-units(4));
```

Comment:
Can just be `@include space.block()` as [the `block()` mixin](https://github.com/libero/storybook/blob/master/src/shared-styles/_space.scss#L32) has `baseline.for-units(4)` as its default `$end` value.